test/integration/create/test_create.py

```diff
@@ -1051,3 +1052,109 @@ def select_data_func():
+
+    app_path = tmp_path / "app1"
+    assert os.path.exists(app_path / "config.yaml")
+    assert os.path.exists(app_path / "instances.yaml")
+
+    # Start app.
+    start_cmd = [tt_cmd, "start", "app1"]
+    instance_process = subprocess.Popen(
+        start_cmd,
+        cwd=tmp_path,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True,
+    )
+    start_output = instance_process.stdout.readlines()
+    for line in start_output:
+        assert "Starting an instance" in line
+    if num_replicas is None:
+        num_replicas = 3
+    instances = [f"replicaset-001-{chr(ord('a')+i)}" for i in range(num_replicas)]
+    pid_files = [os.path.join(tmp_path, "app1", inst) for inst in instances]
+    assert wait_files(10, pid_files)
+
+    # Check status.
+    status_cmd = [tt_cmd, "status", "app1"]
+    status_rc, status_out = run_command_and_get_output(status_cmd, cwd=tmp_path)
+    assert status_rc == 0
+    status_info = extract_status(status_out)
+    for key in status_info.keys():
+        assert status_info[key]["STATUS"] == "RUNNING"
+
+    # def eval_cmd_func(inst, cmd):
+    #     connect_process = subprocess.Popen(
+    #         [tt_cmd, "connect", f"app1:{inst}", "-f-"],
+    #         cwd=tmp_path,
+    #         stdin=subprocess.PIPE,
+    #         stderr=subprocess.STDOUT,
+    #         stdout=subprocess.PIPE,
+    #         text=True,
+    #     )
+    #     connect_process.stdin.write(cmd)
+    #     connect_process.stdin.close()
+    #     connect_process.wait()
+    #     return connect_process.stdout.read()
+
+    # def insert_data_func():
+    #     out = eval_cmd_func("router-001-a", "put_sample_data()")
+    #     return out == "---\n...\n\n"
+
+    # def select_data_func():
+    #     out = eval_cmd_func("router-001-a", "get(1)")
+    #     return out.find("[1, 'Elizabeth', 12]") != -1
+
+    # # Check that data can be inserted and selected.
+    # can_insert = wait_event(60, insert_data_func)
+    # can_select = False
+    # if can_insert:
+    #     can_select = wait_event(60, select_data_func)
+
+    # # Print instances log to find out the reason in case of an assert fall.
+    # for inst in instances:
+    #     with open(os.path.join(tmp_path, "app1", inst, "tt.log")) as f:
+    #         print(inst, f.read())
+
+    # # Stop the vhsard_cluster app.
+    # stop_cmd = [tt_cmd, "stop", "--yes", "app1"]
+    # stop_rc, stop_out = run_command_and_get_output(stop_cmd, cwd=tmp_path)
+    # assert stop_rc == 0
+    # for inst in instances:
+    #     assert re.search(rf"The Instance app1:{inst} \(PID = \d+\) has been terminated.", stop_out)
+
+    # with open(app_path / "router-001-a" / "tt.log") as f:
+    #     print("\n".join(f.readlines()))
+    # # Assert here to be sure that instances are stopped.
+    # assert can_insert, "can not insert data into the vshard cluster"
+    # assert can_select, "can not select data from the vhsard cluster"
```

Comment:
Why is it commented?

Reply:
Oh... it seems I forgot to add the recent changes before pushing.

Reply:
Updated.
Also the new test was introduced that checks defaults for parametrized templates.